packages/ckeditor5-utils/src/version.ts

```diff
@@ -24,138 +24,162 @@ declare global {
 /* istanbul ignore next -- @preserve */
 if ( globalThis.CKEDITOR_VERSION ) {
 	/**
-	 * The best solution to avoid this error is migrating your CKEditor&nbsp;5 instance to
-	 * {@glink updating/nim-migration/migration-to-new-installation-methods new installation methods}.
-	 *
-	 * Mentioned below are predefined builds, which are a deprecated installation method. The solutions
-	 * provided are kept here for legacy support only.
-	 *
-	 * This error is thrown when due to a mistake in how CKEditor 5 was installed or initialized, some
-	 * of its modules were duplicated (evaluated and executed twice). Module duplication leads to inevitable runtime
-	 * errors.
-	 *
-	 * There are many situations in which some modules can be loaded twice. In the worst case scenario,
-	 * you may need to check your project for each of these issues and fix them all.
-	 *
-	 * # Trying to add a plugin to an existing build
-	 *
-	 * If you import an existing CKEditor 5 build and a plugin like this:
-	 *
-	 * ```ts
-	 * import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
-	 * import Highlight from '@ckeditor/ckeditor5-highlight/src/highlight';
+	 * This error is thrown when, due to a mistake in the way CKEditor&nbsp;5 was installed,
+	 * imported, or initialized, some of its modules were evaluated and executed twice.
+	 * Duplicate modules inevitably lead to runtime errors and increased bundle size.
+	 *
+	 * # Check dependency versions
+	 *
+	 * First, make sure that you use the latest version of all CKEditor&nbsp;5 dependencies.
+	 * Depending on the installation method you used, you should check the versions of the `ckeditor5`,
+	 * `ckeditor5-premium-features`, or `@ckeditor/ckeditor5-<NAME>` packages. If you cannot update
```

Comment:
I used `@ckeditor/ckeditor5-<NAME>` instead of `@ckeditor/ckeditor5-*` because paths including `*/` would close the JavaScript comment. If I added an escape character before the `/` (`*\/`), then the import example was invalid.